### ISSUE TITLE
[vite:dep-scan] The plugin "vite:dep-scan" was triggered by this import

### ISSUE THREAD

**Issue by unauthorized-401** [readyplayerme/rpm-react-sdk#2]
```
import { AvatarCreator, EditorConfig } from "@readyplayerme/rpm-react-sdk";

<AvatarCreator
  subdomain="demo"
/>
```

There's some issue with vite plugin. 

```
[ERROR] Failed to resolve entry for package "@readyplayerme/rpm-react-sdk". The package may have incorrect main/module/exports specified in its package.json. [plugin vite:dep-scan]

    node_modules/esbuild/lib/main.js:1360:21:
      1360 │         let result = await callback({
           ╵                      ^

    at packageEntryFailure (file:///C:/Users/Jiwon/Downloads/Projects/mosmos/client/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:23382:11)
    at resolvePackageEntry (file:///C:/Users/Jiwon/Downloads/Projects/mosmos/client/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:23379:5)
    at tryNodeResolve (file:///C:/Users/Jiwon/Downloads/Projects/mosmos/client/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:23113:20)
    at Context.resolveId (file:///C:/Users/Jiwon/Downloads/Projects/mosmos/client/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:22874:28)
    at Object.resolveId (file:///C:/Users/Jiwon/Downloads/Projects/mosmos/client/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:42847:46)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async resolve (file:///C:/Users/Jiwon/Downloads/Projects/mosmos/client/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:43145:26)
    at async file:///C:/Users/Jiwon/Downloads/Projects/mosmos/client/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:43322:34
    at async requestCallbacks.on-resolve (C:\Users\Jiwon\Downloads\Projects\mosmos\client\node_modules\esbuild\lib\main.js:1360:22)
    at async handleRequest (C:\Users\Jiwon\Downloads\Projects\mosmos\client\node_modules\esbuild\lib\main.js:723:13)

  This error came from the "onResolve" callback registered here:

    node_modules/esbuild/lib/main.js:1279:20:
      1279 │       let promise = setup({
           ╵                     ^

    at setup (file:///C:/Users/Jiwon/Downloads/Projects/mosmos/client/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:43312:19)
    at handlePlugins (C:\Users\Jiwon\Downloads\Projects\mosmos\client\node_modules\esbuild\lib\main.js:1279:21)
    at buildOrContextImpl (C:\Users\Jiwon\Downloads\Projects\mosmos\client\node_modules\esbuild\lib\main.js:968:5)
    at Object.buildOrContext (C:\Users\Jiwon\Downloads\Projects\mosmos\client\node_modules\esbuild\lib\main.js:776:5)
    at C:\Users\Jiwon\Downloads\Projects\mosmos\client\node_modules\esbuild\lib\main.js:2172:68        
    at new Promise (<anonymous>)
    at Object.context (C:\Users\Jiwon\Downloads\Projects\mosmos\client\node_modules\esbuild\lib\main.js:2172:27)
    at Object.context (C:\Users\Jiwon\Downloads\Projects\mosmos\client\node_modules\esbuild\lib\main.js:2012:58)
    at prepareEsbuildScanner (file:///C:/Users/Jiwon/Downloads/Projects/mosmos/client/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:43097:26)

  The plugin "vite:dep-scan" was triggered by this import

    src/components/ComputerDialog.tsx:1:44:
      1 │ ... AvatarCreator, EditorConfig } from "@readyplayerme/rpm-react-sdk";
```

Can I import ```@readyplayerme/rpm-react-sdk``` and check with vite?
Please advise me.

**Comment by rk132** [comment]
Hey, there's issues with esm modules right now. looking into it.

**Comment by rk132** [comment]
This is fixed in the [0.1.4](https://www.npmjs.com/package/@readyplayerme/rpm-react-sdk/v/0.1.4). If the issue persists, feel free to let us know.
